client/components/FormattingBar/controlComponents/ControlsLink.tsx

```diff
@@ -71,9 +346,35 @@ const ControlsLink = (props: Props) => {
 				// @ts-expect-error ts-migrate(2322) FIXME: Type 'MutableRefObject<undefined>' is not assignab... Remove this comment to see the full error message
 				inputRef={inputRef}
 			/>
-			<AnchorButton small minimal title="Visit" icon="share" href={href} target="_blank" />
-			<Button small minimal title="Remove" icon="disable" onClick={activeLink.removeLink} />
+			<div>
+				<AnchorButton
+					small
+					minimal
+					title="Options"
+					icon="chevron-down"
+					onClick={() => setIsOpen(!isOpen)}
+				/>
+				<Button
+					small
+					minimal
+					title="Remove"
+					icon="disable"
+					onClick={activeLink.removeLink}
```

Comment:
The onclick for this should remove the connection as well. May not be important just yet but a quick add on to this should be alright